homeassistant/components/bluetooth/passive_update_processor.py

```diff
@@ -374,6 +374,27 @@ def _async_handle_bluetooth_event(
             self.logger.exception("Unexpected error updating %s data", self.name)
             return
 
+        self._process_update(update, was_available)
+
+    @callback
+    def async_manual_update(self, update: _DataT) -> None:
```

Comment:
Probably should name this `async_set_updated_data` to match DUC

https://github.com/home-assistant/core/blob/6737c51fcab133c767b649bb69e6b12a267c8f25/homeassistant/helpers/update_coordinator.py#L500